### PR TITLE
Fixed missing dependency on `chai`.

### PR DESCRIPTION
**Pull request by loic**

### CODE REVIEW

**Comment by loic** [comment]
While we are on the topic of `package.json`, it would be nice to fix the following warnings:
```
npm WARN The package builtin-modules is included as both a dev and production dependency.
npm WARN The package express is included as both a dev and production dependency.
npm WARN The package find-requires is included as both a dev and production dependency.
npm WARN The package glob is included as both a dev and production dependency.
npm WARN The package lodash.flatten is included as both a dev and production dependency.
npm WARN The package lodash.uniq is included as both a dev and production dependency.
npm WARN The package pouchdb-show is included as both a dev and production dependency.
npm WARN The package pouchdb-update is included as both a dev and production dependency.
npm WARN The package pouchdb-validation is included as both a dev and production dependency.
```

I guess the `pouch-*` should be removed from `testDependencies` and the rest depending on usage?

**Comment by loic** [comment]
cc @marten-de-vries @gr2m.

File: package.json [hunk 1]
```diff
@@ -24,7 +24,6 @@
     "couchdb-harness": "*",
     "couchdb-log-parse": "^0.0.4",
     "couchdb-objects": "^1.0.0",
-    "couchdb-eval": "^1.0.0",
```
Comment (loic) [inline package.json]: This one is duplicated line 23 in the same `dependencies` section.

**Comment by loic** [comment]
We have a bootstrapping problem here. `npm install` and therefore the test suite tests against released npm packages (released before they moved to the monorepo), but to get everything working we need many sub-projects to update their dependencies.

I think we should have find a way to test the monorepo against the sub-projects it includes in the same git checkout.

**Comment by loic** [comment]
> I think we should have find a way to test the monorepo against the sub-projects it includes in the same git checkout.

I did that in https://github.com/newlogic42/pouchdb-server/pull/2/commits/29883768ae98015172a73b87be858f1fbc3f4aa2 and I finally got a running test suite on my local machine (albeit with a dependency on my patched version of PouchDB due to #6793).

The full list of fixes: https://github.com/newlogic42/pouchdb-server/pull/2/.

Sadly it seems Travis doesn't like this kind of relative references in package.json, I'm not sure how to proceed at this point.

**Comment by gr2m** [comment]
I remember running into similar problems. I think the right approach would be to pull all pouch* dependencies used only by pouchdb-server into this monorepo. I’m heads down on another project today & tomorrow, I’ll try to dive into this over the weekend. It takes one big effort to make pouchdb-server a monorepo like Nolan did with pouchdb, but once it’s there, things will be simpler to maintain, so shaakaaaa let’s do this :)

**Comment by loic** [comment]
Upgrading `npm` in the `.travis.yml' helped (https://github.com/newlogic42/pouchdb-server/pull/2/commits/9c2210467851ee56dfc8eb36bcb9ab31b7b72535): https://travis-ci.org/newlogic42/pouchdb-server/jobs/290102100

The dependencies with relative path are now properly installed, however the failures suggest that my fork of `pouchdb` is not kicking in since it normally fixes those `Wrapper method for 'replicate.from' not installed` errors.

**Comment by loic** [comment]
Good news, I finally got a green build on Travis for `pouchdb-server`: https://travis-ci.org/newlogic42/pouchdb-server/builds/290290661

The branch: https://github.com/newlogic42/pouchdb-server/pull/2

Of course we are still blocked until

**Comment by gr2m** [comment]
is this blocked by https://github.com/newlogic42/pouchdb-server/pull/2 now?

**Comment by loic** [comment]
I'm afraid I don't remember... That explicit `chai` dependency doesn't seem to be needed for the newlogic42#2 branch to run. I think we can close for now.

**Pull request closed without merging**